lib/ovirt_metrics/configurator.rb

```diff
@@ -0,0 +1,24 @@
+module OvirtMetrics
+  class Configurator
+    def initialize
+      self.suppress_warnings = false
+      self.connection_specification_name = 'ovirt_metrics' if ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
Just for you, I found the original commit when it was introduced. Rails 0.14.1, 2005 https://github.com/rails/rails/commit/68a322005d367981ba72c6a667c16dd171803869

:D